packages/at_secondary_server/lib/src/verb/handler/abstract_verb_handler.dart

```diff
@@ -128,15 +128,16 @@ abstract class AbstractVerbHandler implements VerbHandler {
   ///  - the connection has insufficient permission for this namespace
   ///    (for example, has "r" but needs "rw" for a delete operation)
   ///  - If enrollment is a part of "global" or "manage" namespace
+  ///  - the connection does not have access to * namespace and key has no namespace
+  /// Use [namespace] if passed, otherwise retrieve namespace from [atKey]. Return false if no [namespace] or [atKey] is set.
   Future<bool> isAuthorized(
-    InboundConnectionMetadata connectionMetadata,
-    String keyNamespace,
-  ) async {
+      InboundConnectionMetadata connectionMetadata, String? atKey,
+      {String? namespace}) async {
```

Comment:
Future<bool> isAuthorized(InboundConnectionMetadata connectionMetadata,
      {required String atKey, String? namespace}) async
      Like this ?

Reply:
I guess it could be optional actually so just {String? atKey, String? namespace}

I'm just always a bit confused by positional parameters which allow null